### PR TITLE
@ANSIBLE_TAGS@ replacement for ansible fixes

### PR DESCRIPTION
**Pull request by mpreisler**
Before:
```
<fix id="package_telnet-server_removed" system="urn:xccdf:fix:script:ansible" complexity="low" disruption="low" strategy="disable">- name: "Ensure telnet-server is removed"
  package:
    name="{{item}}"
    state=absent
  with_items:
    - telnet-server
</fix>

```

After:
```
<fix id="package_telnet-server_removed" system="urn:xccdf:fix:script:ansible" complexity="low" disruption="low" strategy="disable">- name: "Ensure telnet-server is removed"
  package:
    name="{{item}}"
    state=absent
  with_items:
    - telnet-server
  tags:
    - package_telnet-server_removed
    - high
    - CCE-27165-0
</fix>
```

### CODE REVIEW

**Comment by mpreisler** [comment]
Snuck in a bugfix and support for check mode. Hope that's fine.

**Comment by shawndwells** [comment]
Is it possible to put the content within the <fix> tag, vs starting on the same line?
(note the ``-name....``)

`````
<fix id="package_telnet-server_removed" system="urn:xccdf:fix:script:ansible" complexity="low" disruption="low" strategy="disable">- name: "Ensure telnet-server is removed"
  package:
    name="{{item}}"
    state=absent
  with_items:
......
`````

vs
`````
<fix id="package_telnet-server_removed" system="urn:xccdf:fix:script:ansible" complexity="low" disruption="low" strategy="disable">
- name: "Ensure telnet-server is removed"
  package:
    name="{{item}}"
    state=absent
  with_items:
......
`````

File: shared/templates/static/ansible/no_empty_passwords.yml [hunk 1]
```diff
@@ -3,9 +3,11 @@
 # strategy = configure
 # complexity = low
 # disruption = medium
-
 - name: "Prevent Log In to Accounts With Empty Password"
   replace:
     dest: /etc/pam.d/system-auth
     regexp: 'nullok\s*'
     replace: ''
+  tags:
```
Comment (shawndwells) [inline shared/templates/static/ansible/no_empty_passwords.yml]: check_mode: no

Comment (shawndwells) [inline shared/templates/static/ansible/no_empty_passwords.yml]: Almost all the tasks need ``check_mode: no`` ... didn't want to litter the PR with multiple of the same comment.

Comment (mpreisler) [inline shared/templates/static/ansible/no_empty_passwords.yml]: Actually, they don't. I only use check_mode no for tasks with no side effects that gather variables.  That's what it's intended for.

**Comment by mpreisler** [comment]
@shawndwells It is possible to put it within the tag but that changes semantics and the generate fix output would be screwed up.

**Comment by mpreisler** [comment]
@shawndwells @redhatrises If you have more ideas for replacements it's very easy for me to add. I don't want to overdo it because I can't think of a use-case for something like `@RULE_TITLE@`. I only want to add replacements we will actively use.

**Comment by shawndwells** [comment]
@mpreisler elements of the ref tag (NIST, DISA, CUI) should be put into the tags... but I agree... later PR.

**Pull request merged**